tfjs-backend-webgpu/src/kernels/FromPixels_utils/from_pixels_webgpu.ts

```diff
@@ -55,6 +57,28 @@ export class FromPixelsProgram implements WebGPUProgram {
 
   constructor() {
     this.shaderKey = 'fromPixels';
+    this.useWgsl = true;  // getUseWgsl();
+  }
+
+  getUserCodeWgsl(): string {
+    const userCode = `
+    [[binding(1), group(0)]] var src: texture_2d<f32>;
```

Comment:
Is above line the only change between from_pixels_webgpu and from_pixels_import_webgpu?  If yes, I prefer you reuse the shader code.
For example, define a common function like below:
```
export function makeFromPixelsSource()
{
  return `
    ${getMainHeaderStringWgsl(this.workGroupSize)} {
      ${getGlobalIndexStringWgsl(this.workGroupSize)}
      let flatIndexBase = index * uniforms.numChannels;
      let coords: vec3<u32> = getCoordsFromFlatIndex(flatIndexBase);
      let values = textureLoad(src, vec2<i32>(coords.yx), 0);
      for (var i: u32 = 0u; i < uniforms.numChannels; i = i + 1u) {
        var value = values[i];
        var flatIndex = flatIndexBase + i;
        if (flatIndex < uniforms.size) {
          result.numbers[flatIndex] = i32(floor(255.0 * value));
        }
      }
    }
`;
}

getUserCodeWgsl(): string {
    const userCode = `
    [[binding(1), group(0)]] var src: texture_2d<f32>;
    ${makeFromPixelsSource()}
   `;
}

Reply:
Good cleanup! Done.